website/docs/docs/build/materializations.md

```diff
@@ -103,6 +104,45 @@ When using the `table` materialization, your model is rebuilt as a <Term id="tab
     * are only used in one or two downstream models, and
     * do not need to be queried directly
 
+### Materialized View
+
+The `materialized view` materialization allows the creation and maintenance of materialized views
+in the target database. This materialization makes use of the `on_configuration_change` config, which
+aligns with the incremental nature of the namesake database object. This setting tells dbt to attempt to
+make configuration changes directly to the object when possible, as opposed to completely recreating
+the object to implement the updated configuration. Using `dbt-postgres` as an example, indexes can
+be dropped and created on the materialized view without the need to recreate the materialized view itself.
+
+The `on_configuration_change` config has three settings:
+- `apply` (default) &mdash; attempt to update the existing database object if possible, avoiding a complete rebuild
+  - *Note:* if any individual configuration change requires a full refresh, a full refresh be performed in lieu of individual alter statements
+- `continue` &mdash; allow runs to continue while also providing a warning that the object was left untouched
+  - *Note:* this could result in downstream failures as those models may depend on these unimplemented changes
+- `fail` &mdash; force the entire run to fail if a change is detected
+
+Materialized views are implemented following this "drop through" life cycle:
+1. If an object does not exist, create a materialized view
+2. If an object exists, other than a materialized view, that object is dropped and replaced with a materialized view
+3. If `--full-refresh` is supplied, replace the materialized view regardless of changes and the `on_configuration_change` setting
+4. If there are no configuration changes, refresh the materialized view
+5. At this point there are configuration changes, proceed according to the `on_configuration_change` setting
```

Comment:
```suggestion
5. At this point if there are configuration changes, proceed according to the `on_configuration_change` setting
```

Reply:
I'm explicitly calling out that we have identified configuration changes, hence we proceed according to the `on_configuration_change` setting. I can reword that, but "if" would be misleading.